apps/notification/receivers/entry_comment.py

```diff
@@ -89,3 +95,45 @@ def create_entry_commit_text_notification(sender, instance, created, **kwargs):
         )
 
     transaction.on_commit(lambda: send_notifications_for_commit(comment.pk, meta))
+
+
+@receiver(m2m_changed, sender=Lead.assignee.through)
+def lead_assignment_signal(sender, instance, action, **kwargs):
+    pk_set = kwargs.get('pk_set', [])
+    if action == 'post_add' and pk_set:
+        for receiver in pk_set:
+            Assignment.objects.create(
+                content_object=instance,
+                created_for_id=receiver,
+                project=instance.project,
+            )
+
+    if action == 'post_remove' and pk_set:
+        Assignment.objects.filter(lead__id=instance.id).delete()
+
+
+@receiver(m2m_changed, sender=EntryComment.assignees.through)
+def entrycomment_assignment_signal(sender, instance, action, **kwargs):
+    pk_set = kwargs.get('pk_set', [])
+    if action == 'post_add' and pk_set:
+        for receiver in pk_set:
+            Assignment.objects.create(
+                content_object=instance,
+                created_for_id=receiver,
+                project=instance.entry.project,
+            )
+
+    if action == 'post_remove' and pk_set:
+        Assignment.objects.filter(entry__id=instance.id).delete()
```

Comment:
@annshress  I don't think so , as this action will handle the changed assignee that have been changed either from lead or entry comment, since this checks the list of assignees id that comes in `pk_set`. Also added the test case in `notification/tests/test_notification.py` go through that.